### PR TITLE
feat(eslint-config-angular): support member ordering

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring
-   [ ] Code style update
-   [ ] Build or CI related changes
-   [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [ ] No

### CODE REVIEW

**Review by IKatsuba — APPROVED** [review]

File: packages/eslint-config-angular/test/member-ordering-correct.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const rule = '@typescript-eslint/member-ordering';
+
+describe('@tinkoff/eslint-config-angular', () => {
+    describe(rule, () => {
+        const cli = new ESlint.CLIEngine({
+            cwd: path.join(__dirname, '..'),
+            useEslintrc: false,
+            baseConfig: {
+                parserOptions: require('../index.js').parserOptions,
+                overrides: [
```
Comment (SuperOleg39) [inline packages/eslint-config-angular/test/member-ordering-correct.test.js:13]: Aren't these overrides need to be directly in rules/typescript-eslint-plugin?

**Review by SuperOleg39 — CHANGES_REQUESTED** [review]

File: packages/eslint-config-angular/test/member-ordering-correct.test.js (hunk 1 above)
Comment (splincode) [reply packages/eslint-config-angular/test/member-ordering-correct.test.js:13]: Yes, but I only want to test `member ordering` rules in isolation without side effect with another rules

**Review by SuperOleg39 — APPROVED** [review]
LGTM

**Review by SuperOleg39 — APPROVED** [review]
LGTM

**Pull request merged**